packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Live/index.tsx

```diff
@@ -1,56 +1,55 @@
 import React, { FunctionComponent } from 'react';
-import { useOvermind } from 'app/overmind';
 
-import LiveInfo from './LiveInfo';
-import LiveButton from './LiveButton';
+import { useOvermind } from 'app/overmind';
 
 import {
   Description,
+  ErrorDescription,
   WorkspaceInputContainer,
   WorkspaceSubtitle,
-  ErrorDescription,
 } from '../../elements';
+
 import { More } from '../More';
 
+import LiveButton from './LiveButton';
+import LiveInfo from './LiveInfo';
+
 export const Live: FunctionComponent = () => {
   const {
+    actions: {
+      live: {
+        createLiveClicked,
+        onAddEditorClicked,
+        onChatEnabledChange,
+        onFollow,
+        onModeChanged,
+        onRemoveEditorClicked,
+        onSessionCloseClicked,
+        onToggleNotificationsHidden,
+      },
+    },
     state: {
-      isLoggedIn,
+      editor: {
+        currentId,
+        currentSandbox: { owned },
+        isAllModulesSynced,
+      },
       live: {
+        followingUserId,
         isLive,
+        isLoading,
         isOwner,
         isTeam,
-        roomInfo,
         liveUserId,
-        reconnecting,
         notificationsHidden,
-        followingUserId,
-        isLoading,
-      },
-      editor: {
-        isAllModulesSynced,
-        currentSandbox: { owned },
-        currentId,
-      },
-    },
-    actions: {
-      live: {
-        onModeChanged,
-        onAddEditorClicked,
-        onRemoveEditorClicked,
-        onSessionCloseClicked,
-        onToggleNotificationsHidden,
-        onChatEnabledChange,
-        onFollow,
-        createLiveClicked,
+        reconnecting,
+        roomInfo,
       },
+      isLoggedIn,
     },
   } = useOvermind();
-  const hasUnsyncedModules = !isAllModulesSynced;
-
-  const showPlaceHolder = (!isLive && !owned) || !isLoggedIn;
 
-  if (showPlaceHolder) {
```

Comment:
I'd keep the var here in. This has a documentational purpose (what do we show if this is true?).

Reply:
Seems reasonable 🙂